modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -5,21 +5,19 @@
 [id="ols-creating-the-credentials-secret-using-cli_{context}"]
 = Creating the credentials secret using the command line
 
-Create a file that is associated with the secret key used to access the API of your LLM provider.
+Create a file that is associated with the secret key used to access the API of your LLM provider. You can API tokens to authenticate your LLM provider. Additionally, Microsoft Azure supports authentication using Entra ID.
```

Comment:
I think we're missing a verb here no?
`You can **use** API tokens...`

Reply:
done